core/src/main/java/org/mule/runtime/core/api/extension/MuleExtensionModelDeclarer.java

```diff
@@ -747,6 +747,7 @@ private void declareConfiguration(ExtensionDeclarer extensionDeclarer, ClassType
         .withOptionalParameter("defaultErrorHandler-ref")
         .ofType(typeLoader.load(String.class))
         .withExpressionSupport(NOT_SUPPORTED)
+        .withAllowedStereotypes(singletonList(ERROR_HANDLER))
```

Comment:
Why this change? Is this related to another issue?

Reply:
the allowedStereotypes of a String parameter identify it as a reference to a global element. In this case, the parameter was declared as an errorHandler-ref, but the model said nothing about it.

Reply:
This 'reference' logic is used here: https://github.com/mulesoft/mule/pull/7110/files#diff-a9272277dbb0602334e99271758985f7R182